parm/config/config.vrfy

```diff
@@ -142,22 +141,25 @@ fi
 # Cyclone genesis and cyclone track verification
 #-------------------------------------------------
 
-export ens_tracker_ver=v1.1.15.1
+export ens_tracker_ver=v1.1.15.5
```

Comment:
Used the same approach as the original. If it needs to be modified, recommend to do in a separate issue ticket.

Reply:
When we ported to WCOSS2, as @KateFriedman-NOAA indicates, this needs to be set in the version file and not in the current location.
If unable to do this in this PR, then please open an issue.  It will be assigned appropriately.
It is in the scope of this work.